platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygTextView.swift

```diff
@@ -53,12 +56,39 @@ public class WysiwygTextView: UITextView {
 
     override public init(frame: CGRect, textContainer: NSTextContainer?) {
         super.init(frame: frame, textContainer: textContainer)
-        contentMode = .redraw
+        commonInit()
     }
     
     required init?(coder: NSCoder) {
         super.init(coder: coder)
+        commonInit()
+    }
+    
+    private func commonInit() {
         contentMode = .redraw
+        NotificationCenter.default.addObserver(self,
```

Comment:
I wonder how this behaves when multiple textviews are present on screen. Should we maybe check for isFirstResponder?